frontend/lib/src/components/shared/Dropdown/Selectbox.tsx

```diff
@@ -86,6 +86,7 @@ const Selectbox: React.FC<Props> = ({
 }) => {
   const theme: EmotionTheme = useTheme()
   const [value, setValue] = useState<number | null>(propValue)
+  const valueBeforeRemoval = useRef<number | null>(value)
```

Comment:
nit: maybe add a comment to explain why the ref is needed?

Reply:
Good point, I have added a comment!